packages/core/src/styles/utils/_close.scss

```diff
@@ -23,8 +23,4 @@ button.close {
   &:active {
     opacity: 1;
   }
-
-  &:focus {
-    @include include-outline-style-form-fields();
-  }
```

Comment:
This is not needed since there is a native focus reset in the base component styles. Also will be replaced anyways soon with closable button component.